temporal-sdk/src/test/java/io/temporal/internal/statemachines/LocalActivityStateMachineTest.java

```diff
@@ -172,7 +172,8 @@ protected void buildWorkflow(AsyncWorkflowBuilder<Void> builder) {
               RespondActivityTaskCompletedRequest.newBuilder().setResult(result2).build(),
               null,
               null,
-              null);
+              null,
```

Comment:
nit: Constructor overload might be nice for these `Result` types as well, looks like they're frequently constructed w/ a bunch of empty params.